### DISCUSSION TITLE
DECIMAL storage management

### DISCUSSION THREAD

**Discussion by Peter-Jacob** [adesutherland/CREXX#362] [Show and tell]
What needs to be done, for DECIMAL memory management?
- I allocate/re-allocate storage whenever it is needed and store its address in register->decimal_value
- from where it is used to perform the arithmetic operations. 
- I don't have control to release it at the end of the task/procedure.
- I'd like to know if we can add it to a general cleanup process we might already have.

**Comment by adesutherland** [comment]
Yes - I will push a change with the cleanup code
